### ISSUE TITLE
BibTeX: truncate author lists

### ISSUE THREAD

**Issue by ehenneken** [adsabs/export_service#109]
In ADS Classic there is a distinction between `BibTeX reference list` (= `BibTeX` in new ADS) and `BibTeX with abstracts` (= `BibTeX ABS` in new ADS): in `BibTeX reference list` the author list is truncated to at most 10 authors in ADS Classic. We did this because BibTeX often crashes on records with huge author lists.

**Comment by golnazads** [comment]
In the new ADS there is also two of them
/bibtex BibTeX reference list
/bibtexabs BibTeX with abstracts

I did not realize that for reference list the authors should be truncated to at most 10. That is what you are saying, right?

**Comment by golnazads** [comment]
Edwin, I was wondering if we should consider this to be a parameter passed in to the endpoint. This way user has complete control over how many authors they like to display. And we can apply I too both list and abstract one. Any thoughts?

**Comment by golnazads** [comment]
Forgot to mention, the parameter can become part of user preferences. We are actually adding another parameter, please see https://github.com/adsabs/bumblebee/issues/1845, so if you would like to proceed this way we can add one more.

**Comment by ehenneken** [comment]
That's correct, for reference list the authors should be truncated to at most 10. I would go for having the Classic behavior as default, but I like the idea of allowing the user to change that number. I wonder if we should take this number from the application settings (in https://ui.adsabs.harvard.edu/user/settings/application) or have it be a separate number. I'm leaning towards it being a separate number.

**Comment by aaccomazzi** [comment]
Agree in general, but still think we should have a reasonable default set in export service.  This way people who fetch bibtex straight from the API endpoint will get something usable without having to worry about the details.  So I'm suggesting the following:

1. export service has a `maxauthor` default
2. export service endpoint accepts a `maxauthor` parameter passed in 
3. BBB implements some kind of custom setting so that each user can choose to override the default at the application level

I think the order above is significant, we should implement 1 right away, then 2 is rather trivial, and eventually we can do 3 at some point.

**Comment by golnazads** [comment]
and per Edwin `maxauthor` be 10, right?

**Comment by aaccomazzi** [comment]
Yes, we know 10 works and it's also what classic uses.

**Comment by golnazads** [comment]
bibtex BibTeX reference default maxauthor is 10
/bibtexabs BibTeX with abstracts default maxauthor is 0 meaning show all

released to dev on 6/4 and Build #261

this record 2019arXiv190600814P has 45 authors, and now we have
```
@ARTICLE{2019arXiv190600814P,
       author = {{Pastorello}, A. and {Reguitti}, A. and {Morales-Garoffolo}, A. and
         {Cano}, Z. and {Prentice}, S.~J. and {Hiramatsu}, D. and {Burke}, J. and
         {Kankare}, E. and {Kotak}, R. and {Reynolds}, T.},
        title = "{A luminous stellar outburst during a long-lasting eruptive phase first, and then SN IIn 2018cnf}",
      journal = {arXiv e-prints},
     keywords = {Astrophysics - Solar and Stellar Astrophysics, Astrophysics - High Energy Astrophysical Phenomena},
         year = "2019",
        month = "Jun",
          eid = {arXiv:1906.00814},
        pages = {arXiv:1906.00814},
archivePrefix = {arXiv},
       eprint = {1906.00814},
 primaryClass = {astro-ph.SR},
       adsurl = {https://dev.adsabs.harvard.edu/abs/2019arXiv190600814P},
      adsnote = {Provided by the SAO/NASA Astrophysics Data System}
}
```
